openapi/generated/api/processors-api.ts

```diff
@@ -162,12 +166,15 @@ export const ProcessorsApiAxiosParamCreator = function (configuration?: Configur
          * Get the list of Processors of a Bridge instance for the authenticated user.
          * @summary Get the list of Processors of a Bridge instance
          * @param {string} bridgeId 
+         * @param {string} [name] 
          * @param {number} [page] 
          * @param {number} [size] 
+         * @param {Set<ManagedResourceStatus>} [status] 
+         * @param {ProcessorType} [type] 
          * @param {*} [options] Override http request option.
          * @throws {RequiredError}
          */
-        listProcessors: async (bridgeId: string, page?: number, size?: number, options: AxiosRequestConfig = {}): Promise<RequestArgs> => {
+        listProcessors: async (bridgeId: string, name?: string, page?: number, size?: number, status?: Set<ManagedResourceStatus>, type?: ProcessorType, options: AxiosRequestConfig = {}): Promise<RequestArgs> => {
```

Comment:
If we check, coresponding java class, there are parameters `@NotEmpty @PathParam("bridgeId") String bridgeId, @Valid @BeanParam QueryProcessorResourceInfo queryInfo` 

I am not sure if that is an issue of code generation, that `QueryProcessorResourceInfo` is extracted into particular parameters. What do you think?

Reply:
I think everything is fine from the API perspective. `name`, `status`, `type` are all new optional filters that it's possible to pass as query params to get the processors list